### PR TITLE
4403 as a paying customer you have access module for org users should be different

### LINKED ISSUE #4403: "As a paying customer you have access..." module for org users should be different
This should be "Contact the organization admin for support."

In the Organization settings, there should be an “admin email” setting where these users can contact them. We don’t want every student at Stanford to be contacting Carla and Stephanie, that would be a lot! But the org admin is the person who’s paying us, so if they need support they can contact CartoDB HQ.

### PR DESCRIPTION
**Pull request by juanignaciosl**
This closes #4403 . @xavijam , @rafatower CR this, please.

### CODE REVIEW

**Comment by rafatower** [comment]
:+1:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5363/))

**Comment by juanignaciosl** [comment]
@javierarce could you CR this, please? This has already passed the tests in a previous build. If you want to test it it's deployed in ded01.

File: lib/assets/javascripts/cartodb/common/views/support_banner.jst.ejs [hunk 1]
```diff
@@ -32,7 +36,7 @@
         <% } %>
       </h4>
       <p class="SupportBanner-infoDescription">
-        <% if (userType === 'org' || userType === 'client') { %>
+        <% if (userType === 'org' || userType == 'org_admin' || userType === 'client') { %>
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb/common/views/support_banner.jst.ejs]: Can you add the missing equal sign here: ```userType == 'org_admin' ```?

File: lib/assets/javascripts/cartodb/common/views/support_banner.jst.ejs [hunk 2]
```diff
@@ -12,7 +14,7 @@
     <div class="SupportBanner-icon">
       <i class="
         iconFont
-        <% if (userType === 'org') { %>
+        <% if (userType === 'org_admin' || userType == 'org') { %>
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb/common/views/support_banner.jst.ejs]: Can you add the missing equal sign here: ```userType == 'org' ```?

File: lib/assets/javascripts/cartodb/models/user.js [hunk 3]
```diff
@@ -37,6 +37,8 @@ cdb.admin.User = cdb.core.Model.extend({
       this._fetchLayers();
     }
 
+    this.email = ((typeof attrs != 'undefined') && attrs != null && (typeof attrs.email != 'undefined')) ? attrs.email : '';
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb/models/user.js]: We should try to use ```!==``` whenever possible.

**Comment by javierarce** [comment]
:+1: but check the comments about the comparison operators.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5372/))

**Comment by juanignaciosl** [comment]
@javierarce added, thanks!

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5374/))

**Comment by juanignaciosl** [comment]
Retest this, please.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5385/))

**Pull request merged**